tests/test_plugins.py

```diff
@@ -284,7 +284,7 @@ class Model(BaseModel):
         assert Model.model_validate_json('{"a": 2}', context={'c': 2}).model_dump() == {'a': 2}
         # insert_assert(log)
         assert log == [
-            'json enter input={"a": 2} kwargs={\'strict\': None, \'context\': {\'c\': 2}}',
+            "json enter input={\"a\": 2} kwargs={'strict': None, 'context': {'c': 2}}",
```

Comment:
@charliermarsh if you are looking for feedback, I'd think that it would be preferable to use single quotes on the outer level when single quotes are preferred and the string contains both single and double quotes. But I definitely don't care enough to hold off on merging just for this issue 😄

Reply:
Yeah I agree — this one struck me as odd. Gonna look into it.

Reply:
https://github.com/astral-sh/ruff/issues/8264